frontend/src/libs/components/components.ts

```diff
@@ -3,6 +3,7 @@ export { Avatar } from './avatar/avatar.js';
 export { Button } from './button/button.js';
 export { CommentCard } from './comment-card/comment-card.js';
 export { Comments } from './comments/comments.js';
+export { ConfirmArticleDeleteDialog } from './confirm-article-delete-dialog/confirm-article-delete-dialog.js';
```

Comment:
Let's create separate `components` in `pages` folder and move more specific but reusable components for page there